src/borg/archiver/create_cmd.py

```diff
@@ -196,6 +196,9 @@ def create_inner(archive, cache, fso):
                             json_print(basic_json_data(manifest, cache=cache, extra={"archive": archive}))
                         else:
                             log_multi(str(archive), str(archive.stats), logger=logging.getLogger("borg.output.stats"))
+                        print(f"Files status: {fso.stats.files_stats}")
```

Comment:
looks like this would render a dict?

Reply:
see the few lines of code right above what you added: the output needs to be either json or human readable text.